### DISCUSSION TITLE
great work, good idea

### DISCUSSION THREAD

**Discussion by Maestro8484** [monoapp3/Laterna#1] [General]
i like what you've done.  how can i buy one or whats easiest route to build something similar for a person with many WLED projects?

**Comment by monoapp3** [comment]
Hi Maestro, sorry for late answer. 
If you are interested on our boards, you can find the links in each individual page.

Currently due to Holidays and after long COVID restrictions the team is out of office. => Guys traveling around :)
Please have a look in 2 weeks time, then you should be able to find available stock.

If you have any other comment or idea let us know

**Comment by Henry-Jones** [comment]
Hello,

there seems to be stock available now, but how can I order from Germany ? Aliexpress states that they cannot sent this product to Germany.

**Reply by monoapp3** [reply]
Hello Henry-Jones,

we just realized about this before yesterday as well.  Currently checking what is the issue in our shop configuration.
Thanks to mention this!

Will provide feedback once issue solved. We do have Stock also for other boards

**Reply by Henry-Jones** [reply]
Is this ever going to be resolved ? I just checked out of curiosity and the fact still remains.
